### ISSUE TITLE
Enum empty in plantuml

### ISSUE THREAD

**Issue by Eldynn** [bafolts/tplant#16]
Thanks for this :) !

I've an issue with enum when I use tplant the enum is empty.

> cat MySuperEnum.ts
```
export enum MySuperEnum {
  AWESOME,
  SUPER,
  COOL
}
```
> npx tplant --version
```
2.0.3
```
> npx tplant --input MySuperEnum.ts
```
@startuml
enum MySuperEnum {
}
@enduml
```

**Comment by bafolts** [comment]
This might be fixed in `master` but hasn't been released in 2.0.4 yet.

**Comment by Eldynn** [comment]
I test that when I can, and report here the result.

**Comment by bafolts** [comment]
I am seeing this in `master` as well. It is surprising since there is a unit test that an enum runs through. A fix for this can go in 2.0.4

**Comment by bafolts** [comment]
Disregard, having never used `npx` I didn't realize it was pulling down `2.0.3`. Enum does work fine in `master` but hasn't been published as `2.0.4` to npm yet. This will work once `2.0.4` is published which I will do tonight.

**Comment by bafolts** [comment]
This was published to `2.0.4` and should now work.
